plugin/trino-snowflake/src/main/java/io/trino/plugin/snowflake/SnowflakeConfig.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.snowflake;
+
+import io.airlift.configuration.Config;
+
+import java.util.Optional;
+
+public class SnowflakeConfig
+{
+    private String account;
+    private String database;
+    private String role;
+    private String warehouse;
+    private Boolean timestampNoTimezoneAsUTC;
+    private String httpProxy;
```

Comment:
These config properties are not tested. Please remove them from the initial PR.

Reply:
Removed

Reply:
Reminder.